hooli_data_eng/resources/__init__.py

```diff
@@ -58,6 +59,7 @@
 
 resource_def = {
     "LOCAL": {
+        "power_bi": power_bi_workspace,
```

Comment:
I wonder if there is something we can do here to mock it out so folks who clone and run this locally without the powerBI creds still work?

(Speaking as someone who will want to be able to run the demo locally without worrying if I have sourced the power BI creds correctly)